app/src/main/java/fr/free/nrw/commons/auth/login/LoginClient.kt

```diff
@@ -106,6 +107,35 @@ class LoginClient(private val loginInterface: LoginInterface) {
         })
     }
 
+    fun doLogin(
+        username: String,
+        password: String,
+        twoFactorCode: String? = null,
+        userLanguage: String = Locale.getDefault().language,
+        loginCallback: LoginCallback
+    ) {
+        getLoginToken().enqueue(object :Callback<MwQueryResponse?>{
```

Comment:
If no-one outside of the `LoginClient` is calling the `getLoginToken()` method, can you make it _private_?

Reply:
Sure!! I made the changes in the commit.